assets/site-app/resources/images/site/start.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/sh
```

Comment:
Hmm, I'm just noticing this but why do we have the same `start.sh` script twice - in site-app/images/site (above) and here? Do we need both?

Reply:
I think the one in `resources/images/site` not in-use - will remove.